homeassistant/components/mqtt/config_integration.py

```diff
@@ -169,9 +158,11 @@
 CONFIG_SCHEMA_ENTRY = vol.Schema(
```

Comment:
In a follow-up, I think we can drop the schema? There's no reason to validate config entry options / data?

Reply:
The schema adds default values. If we want to drop the schema we need to change the config flow to save basic broker settings with broker `CONFIG_SCHEMA_ENTRY`. So I don't see an option to drop it yet.

Reply:
I understand, but there's no need to add default values via a schema, just use dict.get instead and pass the wanted default value.

Reply:
So let's implement this in a follow up PR